### DISCUSSION TITLE
How can I use delta's side by side feature when doing `git add -p`?

### DISCUSSION THREAD

**Discussion by neighthan** [dandavison/delta#1390] [Q&A]
When I run `git diff`, I have delta's side by side view, but when I do `git add -p`, I get the one-column view instead. Is there a way to set it up to have the side by side view for `git add -p` as well?

Here is the relevant portion of my git config:

```
[core]
        pager = delta
[interactive]
        diffFilter = delta --color-only
[delta]
        navigate = true
        side-by-side = true
[merge]
        conflictstyle = diff3
[diff]
        colorMoved = default
```

I have `delta --version` 0.15.1 and `git --version` 2.34.1

**Comment by dandavison** [comment]
Hi @neighthan, I don't believe so - git requires that the lines of the displayed patch are in 1:1 correspondence with those of the original (unified) patch. Which makes sense, since git needs to know how to "chop" the patch into smaller pieces. But perhaps in theory there could be a solution with new work git-side and some way for the diff filter to distinguish between the patch for application vs the displayed view.
